src/Main.java

```diff
@@ -273,6 +294,42 @@ public static void task2(){
             System.out.println("Данные Ф. И. О. сотрудника — " + fullNameTrue);
         }
     }
+    public static void calculateLeapYear(int yearNow){
+        short startYear = 1584;
+        if (yearNow > startYear && yearNow % 4 == 0 && yearNow % 100 != 0 || yearNow > startYear && yearNow % 400 == 0) {
```

Comment:
```suggestion
        if (yearNow > startYear && yearNow % 4 == 0 && yearNow % 100 != 0 || yearNow % 400 == 0) {
```
зачем дублировать проверку года?